File: Python/1066.py
a = []
j = 0
k = 0
p = 0
l = 0
for i in range(5):
    x = int(input())
    if x % 2 == 0:
        j += 1
    if x % 2 == 1:
        k += 1
    if x > 0:
        p += 1
    if x < 0:
        l += 1

print(j, 'valor(es) par(es)')
print(k, 'valor(es) impar(es)')
print(p, 'valor(es) positivo(s)')
print(l, 'valor(es) negativo(s)')

File: URI/1066.js
let input = require ('fs').readFileSync('./dev/stdin/file.txt','utf8')
let = lines = input.split('\n')
let dividir = 0
let contadorDePares = 0
let contadorDeImpares = 0
let contadorDePositos = 0
let contadorDeNegativos = 0
for (let i = 0; i <5; i++) {
    let valor = +lines.shift()
    dividir = valor%2
    if((dividir===0)||(valor ===0)){
        contadorDePares++
    }else{
        contadorDeImpares ++
    }
    if (valor>0){
        contadorDePositos++
    }else if (valor < 0){
        contadorDeNegativos++
    }
}
console.log(`${contadorDePares} valor(es) par(es)`)
console.log(`${contadorDeImpares} valor(es) impar(es)`)
console.log(`${contadorDePositos} valor(es) positivo(s)`)
console.log(`${contadorDeNegativos} valor(es) negativo(s)`)
